### PR TITLE
MM-45313_Cannot access the create account button on safari for iOS

### PR DESCRIPTION
**Pull request by julmondragon**
#### Summary
In Safari for iPhone, the Signup page now allows to scroll, so the Create Account button can be accessed.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-45313
https://github.com/mattermost/mattermost-server/issues/20536

#### Screenshots
https://user-images.githubusercontent.com/79058848/199356814-2882ab5f-35ca-4c2f-88fa-21e415ad4fe5.mp4

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/header_footer_route/header_footer_route.scss [hunk 1]
```diff
@@ -20,6 +20,8 @@
         background-position: center;
         background-repeat: no-repeat;
         background-size: cover;
+        // Hack for issue in iOS Safari with scrolling
```
Comment (pvev) [inline components/header_footer_route/header_footer_route.scss:23]:
```suggestion
        // MM-45313 - Hack for issue in iOS Safari with scrolling
```

**Review by pvev — APPROVED** [review]
LGTM! thanks @julmondragon

**Review by marianunez — APPROVED** [review]

**Review by stevemudie — APPROVED** [review]
Worked as expected on Safari on iOS 16.1

**Pull request merged**